src/main/java/org/broadinstitute/hellbender/tools/exome/TargetArgumentCollection.java

```diff
@@ -58,7 +58,7 @@
      * @throws IllegalArgumentException if {@code defaultTargetFileSupplier} is {@code null}.
```

Comment:
not sure if you wanted to change "defaultTargetFileSupplier -> defaultTargetsFileSupplier" everywhere, but you missed a few (4) cases.  also might want to change null message below accordingly.